Review Assignment/src/assets/scripts/about.js

```diff
@@ -0,0 +1,166 @@
+    localStorage.setItem(key, value);
+  }
+
+  function getData(key) {
+    var value = localStorage.getItem(key);
+    return value;
+  }
+
+  function storeData(fullName, gender, birthDate, mStatus, location, occupation, skills, job) {
+    var newData =  {
+      'name': fullName,
+      'gender': gender,
+      'birthDate': birthDate,
+      'status': mStatus,
+      'location': location,
+      'occupation': occupation,
+      'skill': skills,
+      'job': job
+    };
+
+    localStorage.setItem('userData', JSON.stringify(newData));
+  }
+
+  function personData() {
+    const userData = JSON.parse(localStorage.getItem('userData'));
+    return userData;
+  }
+
+  function defaultUserInfo() {
+    var userData = personData();
+    if (userData == null) {
+      let dataUrl = '../assets/data/about.json';
+
+      $.get({url: dataUrl, type: type, success: function(userDetails)  {
+        storeData(userDetails.name, userDetails.gender, userDetails.birthDate, userDetails.status, userDetails.location, userDetails.occupation, userDetails.skills, userDetails.job);
+        updatePersonalInfo();
+      }});
+    }
+  }
+
+  //Default values of about info and work form deatails
+  function personalInfo() {
+    var userInfo = personData();
+    let userName = userInfo.name;
+    let gender = userInfo.gender;
+    let bday = userInfo.birthDate;
+    let status = userInfo.status;
+    let currLocation = userInfo.location;
+
+    $('.name-input').val(userName);
+    $('.gender-input').val(gender);
+    $('.birth-date-input').val(bday);
+    $('.merital-status-input').val(status);
+    $('.current-location-input').val(currLocation);
+
+    $('.user-name').text(userName)
+    $('.user-fullname').text(userName);
+    $('.profile-name').text(userName);
+    $('.name').text(userName);
+    $('.gender').text(gender);
+    $('.birth-date').text(bday);
+    $('.merital-status').text(status);
+    $('.location').text(currLocation);
+  }
+
+  function professionalInfo() {
+    var userInfo = personData();
+    let occupation = userInfo.occupation;
+    let skill = userInfo.skill;
+    let job = userInfo.job;
+
+    $('.occupation-input').val(occupation);
+    $('.skills-input').val(skill);
+    $('.job-input').val(job);
+
+    $('.occupation').text(occupation);
+    $('.skills').text(skill);
+    $('.job').text(job);
+    $('.profile-job').text(occupation);
+  }
+
+  function submitEventData() {
+    var fullName = $('.name-input').val();
+    var gender = $('.gender-input').val();
+    var birthDate = $('.birth-date-input').val();
+    var mStatus = $('.merital-status-input').val();
+    var location = $('.current-location-input').val();
+    var occupation = $('.occupation-input').val();
+    var skills = $('.skills-input').val();
+    var job = $('.job-input').val();
+
+    storeData(fullName, gender, birthDate, mStatus, location, occupation, skills, job);
+  }
+
+  //display edit form data into about section
+  function pencilClickEvents() {
```

Comment:
`pencilClickEvents` give some different function name or may be contextual to the action.